src/style.css

```diff
@@ -399,10 +405,12 @@ li.CodeMirror-hint-active {
 	/*color: white;*/
 }
 #demo-frame {
-	border: 0;
+	border: 0; padding: 0; margin: 0;
```

Comment:
Please add these to their own lines and not inline with the border styling.
This will be more consistent with the rest of the CSS.

Reply:
Surely you realise this PR is well over a year old? The code in this PR is terrible and should never be merged.